units/commit_queue.go

```diff
@@ -118,6 +118,18 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		j.AddError(errors.Errorf("no commit queue found for id %s", j.QueueID))
 		return
 	}
+
+	front, hasItem := cq.Next()
+	if hasItem {
+		grip.Info(message.Fields{
+			"source":       "commit queue",
+			"job_id":       j.ID(),
+			"item_id":      front.Issue,
```

Comment:
can `front` be nil if `hasItem` is `true`?

Reply:
it's not a pointer. hasItem is used to tell if the return value is actually an entry, or just an empty struct